src/PKSim.Presentation/DTO/Parameters/ParameterDTO.cs

```diff
@@ -81,11 +82,11 @@ public virtual double Value
             //Permeability cannot be read in compound as references cannot be resolved
             //We return zero for now so that the rules are not showing a broken state (>=0). 
             //We probably need to update the rules framework to not evaluate on NaN.
-            if (!success)
-               return 0;
+            if (success)
+               return result;
             
-            return result;
-
+            //the returned value will be based on the parameter. For Initial concentration, we will return NaN, otherwise 0
+            return Parameter.IsNamed(INITIAL_CONCENTRATION) ? result : 0;
```

Comment:
that simple... We nee dto resturn nan so that it is displayed as <enter a value>